solutions/cses-1662.mdx

```diff
@@ -80,5 +81,52 @@ int main() {
 	cout << ans << endl;
 }
 ```
+</CPPSection>
+	
+<JavaSection>
+	
+```java
+
+import java.util.*;
+import java.io.*;
+	
+// @Author Brad Ma
```

Comment:
idt this is necessary? your name is already in the authors list...

Reply:
i just saw the guy above do this so I did it, but I will remove it